staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/oidc/oidc.go

```diff
@@ -914,36 +914,65 @@ func (a *jwtAuthenticator) getUID(ctx context.Context, c claims, claimsUnstructu
 	return evalResult.EvalResult.Value().(string), nil
 }
 
-func (a *jwtAuthenticator) getExtra(ctx context.Context, claimsUnstructured *unstructured.Unstructured) (map[string][]string, error) {
-	if a.celMapper.Extra == nil {
-		return nil, nil
-	}
+func (a *jwtAuthenticator) getExtra(ctx context.Context, c claims, claimsUnstructured *unstructured.Unstructured) (map[string][]string, error) {
+	extra := make(map[string][]string)
 
-	evalResult, err := a.celMapper.Extra.EvalClaimMappings(ctx, claimsUnstructured)
-	if err != nil {
+	if credentialID, err := getCredentialID(c); err != nil {
 		return nil, err
```

Comment:
I'm not sure we should turn a non-string `jti` claim into an authentication failure here...

Reply:
should we just add a warning with the error instead?

Reply:
is it useful for the user *using* a credential to get a warning that their token's jti claim is weird? it doesn't seem like that user can do anything about it

Reply:
Would these logs help the admin understand why the apiserver is failing to set the extra info during authentication for provider1 but not for provider2, even though the jti claim exists in both tokens?

Actually, since the only failure cause is a non-string value, maybe just adding a note in the docs would be enough to clarify this without needing the error log.

Reply:
I don't think we want to return the warning to the user, and I don't think we want to log on every request by every credential with this issue... I think documenting the claim will be included if it is present and a string value is sufficient.

Reply:
Agreed! I've pushed a change with this.